src/gluonts/ev/aggregations.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass
+from typing import List, Optional, Union
+
+import numpy as np
+
+
+@dataclass
+class Aggregation:
+    def step(self, values: np.ndarray) -> None:
+        raise NotImplementedError
+
+    def get(self) -> np.ndarray:
+        raise NotImplementedError
+
+
+@dataclass
+class Sum(Aggregation):
+    axis: Optional[int] = None
+    partial_result: Optional[Union[List[np.ndarray], np.ndarray]] = None
+    initial_step: bool = True
```

Comment:
This looks weird. `initial_steps` should only be none iff `partial_result` is also none.

Reply:
There was a check for `if self.partial_result is None:` before.

Adding `initial_step` was introduced to make it more readable (see [here](https://github.com/awslabs/gluonts/pull/2267#discussion_r999370252)). We can revert it again, if you like.